index.html

```diff
@@ -230,13 +231,13 @@ <h2 class="project-name">Multi-Post Stories</h2>
             </ul>
           </div>
 
-          <button type="button" class="project-button">See Project</button>
+          <button type="button" class="project-button project-button1" id="button1">See Project</button>
         </div>
       </div>
       <!--project template 1 ends here-->
 
       <!--project template 2-->
-      <div class="project-reverse">
+      <div class="project-reverse project">
```

Comment:
- [ ] Based on requirements all the content of popup modal is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie: eg:
````javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
````
Please create an object and make the modal popup data populated with the data from the projects array. 😉



If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty